### PR TITLE
Create DevTools Services

### PR DESCRIPTION
**Pull request by jasonLaster**
### Summary of Changes

Services maps to Services.jsm in MC and we want to be able to use the real module in MC. 

We currently do the mapping in [webpack.config.devtools.js][wc]. You can see how [vendors.js][v] pulls in devtools-modules, but excludes Services.

It might be nice to simplify the process a bit by creating a package for it in core, we can require it directly in the debugger and reps and then at release time map it to the real thing. I think this system is marginally better, but perhaps not needed...

[wc]: https://github.com/devtools-html/devtools-core/blob/master/packages/devtools-launchpad/webpack.config.devtools.js#L28-L32
[v]: https://searchfox.org/mozilla-central/source/devtools/client/debugger/new/dist/vendors.js

### CODE REVIEW

File: packages/devtools-services/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Services
+
+[![Npm version](https://img.shields.io/npm/v/devtools-services.svg)](https://npmjs.org/package/devtools-services)
+
+Shims [Services.jsm](https://searchfox.org/mozilla-central/source/toolkit/modules/Services.jsm)
+
+* `prefs`
+* `Services`
```
Comment (nchevobbe) [inline packages/devtools-services/README.md]: I think we only export the method of `Services` and not `{Services, prefs}`

**Comment by nchevobbe** [comment]
and circle is ✅

File: packages/devtools-services/README.md (hunk 1 above)
Comment (jasonLaster) [reply packages/devtools-services/README.md]: that makes sense

**Comment by jasonLaster** [comment]
merge at will

**Pull request merged**